### PR TITLE
issue #5 create script for instancing containers

### LINKED ISSUE #5: create script for instancing containers

### PR DESCRIPTION
**Pull request by PabloAlice**
Fix #5

### CODE REVIEW

File: lib/generate/container-components.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = function components({ name }) {
```
Comment (beogip) [inline lib/generate/container-components.js]: export using es6

File: lib/generate/container-redux-components.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+module.exports = function components({ name }) {
```
Comment (beogip) [inline lib/generate/container-redux-components.js]: export using es6

File: lib/generate/index.js [hunk 3]
```diff
@@ -4,41 +4,52 @@ const regenerateCustomIndex = require('./regenerate-custom-index');
...
-      return {
-        absPath: path.resolve(customFolder, `${type}/${route}/${name}.js`),
-        content: generators[type]({ name, route }),
-      };
-    default:
-      throw new Error(`Type ${type} is not supported`);
+      extension = 'js';
+      break
   }
+      if(extension){
```
Comment (beogip) [inline lib/generate/index.js]: change this lines with a early throw error

```javascript
if(!extension){
  throw new Error(`Type ${type} is not supported`);
}

ensurePathExist(path.resolve(customFolder, `${initFolder}/${route}`));
return {
  absPath: path.resolve(customFolder, `${initFolder}/${route}/${name}.${extension}`),
  content: generators[type]({ name, route })
}
```

File: lib/generate/container-components.js (hunk 1 above)
Comment (PabloAlice) [inline lib/generate/container-components.js]: this project doesn't have es6's export statement

File: lib/generate/container-redux-components.js (hunk 2 above)
Comment (PabloAlice) [inline lib/generate/container-redux-components.js]: the same here

**Pull request merged**